pkg/granted/tokens.go

```diff
@@ -0,0 +1,95 @@
+package granted
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/common-fate/granted/pkg/config"
+	"github.com/common-fate/granted/pkg/credstore"
+	"github.com/common-fate/granted/pkg/testable"
+	"github.com/urfave/cli/v2"
+)
+
+var DefaultTokenCommand = cli.Command{
+	Name:        "token",
+	Usage:       "Functionality to make ",
+	Subcommands: []*cli.Command{&TokenListCommand, &ClearTokensCommand, &ClearAllTokensCommand},
+	Action: func(c *cli.Context) error {
+		//return the default browser that is set
+		conf, err := config.Load()
+		if err != nil {
+			return err
+		}
+		fmt.Fprintf(os.Stderr, "Granted is using %s. To change this run `granted browser set`.\n", conf.DefaultBrowser)
+
+		return nil
+	},
+}
+
+var TokenListCommand = cli.Command{
+	Name:  "list",
+	Usage: "Remove all saved tokens from keyring and delete all granted configuration",
```

Comment:
this comment needs to reflect the command, looks like it was copied from the clear command